components/profile_popover/index.js

```diff
@@ -21,6 +29,34 @@ import {getSelectedPost, getRhsState} from 'selectors/rhs';
 
 import ProfilePopover from './profile_popover.jsx';
 
+function hasManageChannelMemberPermission(state) {
+    const myMemberships = getMyChannelMemberships(state);
+    const teamId = getCurrentTeamId(state);
+    const permissions = [
+        Permissions.MANAGE_PUBLIC_CHANNEL_MEMBERS,
+        Permissions.MANAGE_PRIVATE_CHANNEL_MEMBERS,
+    ];
+
+    for (const channelId in myMemberships) {
```

Comment:
A more explicit wat that no require the "hasOwnProperty" check would be something like:
```js
for (const channelId of Object.keys(myMemberships)) {
    ...
}
```